test/index.test.js

```diff
@@ -441,7 +441,9 @@ describe('index.js', () => {
                     property = expose.property.slice(0, (expose.endpoint.length + 1) * -1);
                 }
 
-                const toZigbee = device.toZigbee.find((item) => item.key.includes(property));
+                const toZigbee = device.toZigbee.find(
```

Comment:
Why was this change needed?

Reply:
This change has the same purpose as the one we did here : https://github.com/Koenkk/zigbee2mqtt/blame/bc6eccf6152c4cd91e75309f7ea0b8ad4e8835d4/lib/extension/publish.ts#L263

Let's take an example : If you have for instance a device made of a winndowClosures cluster (Ex: A garage door) and a onOff cluster (Ex: A garage light), then you have here a mix up between toZigbee converter because `find` function stops at the first key `state` found if we don't use the additional endpoint criteria. This is also explained in issue https://github.com/Koenkk/zigbee2mqtt/issues/24352.

Today, with the removal of my `GADOLOCK` device, no false positive error will be triggered (even with all others thousand devices! ).

I hope I am clear enough. Please advise if you prefer keep this change preventively or remove it because considered not relevant. At my level (a simple z2m user), I will simply follow your recommendation 🙂